src/pg/sql/45_observatory_mvt.sql

```diff
@@ -218,16 +218,65 @@ BEGIN
 END
 $$ LANGUAGE plpgsql;
 
-CREATE OR REPLACE FUNCTION cdb_observatory.OBS_GetMCDOMVT(z INTEGER, x INTEGER, y INTEGER,
-geography_level TEXT,
-do_measurements TEXT[],
-mastercard_measurements TEXT[],
-shoreline_clipped BOOLEAN DEFAULT True,
-optimize_clipping BOOLEAN DEFAULT False,
-simplify_geometries BOOLEAN DEFAULT False,
-area_normalized BOOLEAN DEFAULT False,
-mastercard_category TEXT DEFAULT 'Total Retail',
-extent INTEGER DEFAULT 4096, buf INTEGER DEFAULT 256, clip_geom BOOLEAN DEFAULT True)
+DROP TABLE IF EXISTS cdb_observatory.OBS_CachedMeta;
+CREATE TABLE cdb_observatory.OBS_CachedMeta(
+  z INTEGER,
+  parameters TEXT,
+  num_timespans INTEGER,
+  num_scores INTEGER,
+  num_target_geoms INTEGER,
+  result JSON,
+  PRIMARY KEY (z, parameters, num_timespans, num_scores, num_target_geoms)
+);
+
+CREATE OR REPLACE FUNCTION cdb_observatory.OBS_RetrieveMeta(
```

Comment:
I haven't named it like that because the function gets the info either from meta or the cached table. If we change the mechanism to get the cached data for a more elaborate one, we can add a `GetCachedMeta` function that gets called from this function to do the job